src/System.Composition.sln

```diff
@@ -0,0 +1,97 @@
+﻿
```

Comment:
I'm not a fan of keeping this directly in \src, all the other solution files are inside a subfolder and it looks much cleaner. Can we move this to \src\System.Composition?